raiden-ts/src/transport/utils.ts

```diff
@@ -107,3 +108,13 @@ export function parseCaps(caps?: string | null): Caps | undefined {
 export function getCap<C extends Capabilities>(caps: Caps | undefined | null, cap: C): Caps[C] {
   return caps?.[cap] ?? CapsFallback[cap];
 }
+
+/**
+ * Return addresses sorted in lexical order
+ *
+ * @param addresses - Addresses to sort
+ * @returns Addresses sorted in lexical order
+ */
+export function getSortedAddresses<A extends [Address, ...Address[]]>(...addresses: A) {
```

Comment:
Some simple tests for this would be nice. I remember some there was some bug in this function in the PC.

Reply:
Done. Rebasing.